pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "google_benchmark"
+description = "A library to benchmark code snippets."
+requires-python = ">=3.8"
+license = {file = "LICENSE"}
+keywords = ["benchmark"]
+
```

Comment:
maybe also readme pointing to the github readme?

Reply:
The readme is dynamic, see the last `[tool.setuptools]` block. Not sure what happens if it is specified twice, but I can also add it to the `project` part.

Reply:
the project readme should be a link to the existing markdown file in github aiui

Reply:
Hmm, I don't think so: Here in the docs, it says that if it is a string, it needs to be a path relative to the `pyproject.toml`: https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#readme

However, also in that docs section, it appears that from an `.md` extension, it automatically assumes `content-type = text/markdown`, so the dynamic part can go away and be replaced with the static `readme = "README.md"` field. What do you think?

Reply:
i probably read it wrong then :)